### PR TITLE
feat: add 5Y filter to bitcoin price chart

### PR DESCRIPTION
**Pull request by eimolina**
My first contribution.

### CODE REVIEW

File: app/components/price-graph/price-graph.tsx [hunk 1]
```diff
@@ -144,6 +149,8 @@ export const PriceGraph: ComponentType = ({
         return translate("PriceScreen.thisMonth")
       case GraphRange.ONE_YEAR:
         return translate("PriceScreen.thisYear")
+      case GraphRange.FIVE_YEARS:
+        return translate("PriceScreen.lastFiveYears")
```
Comment (dolcalmi) [inline app/components/price-graph/price-graph.tsx:153]: does this require update galoy-client first, right?

Comment (dolcalmi) [reply app/components/price-graph/price-graph.tsx:153]: @eimolina https://github.com/GaloyMoney/galoy-client/pull/50 was merged and published.. Please update package with the new version

Comment (eimolina) [reply app/components/price-graph/price-graph.tsx:153]: image

Comment (eimolina) [reply app/components/price-graph/price-graph.tsx:153]: I just updated the galoy-client library in the dependencies.

**Review by dolcalmi — APPROVED** [review]

**Review by daviroo — APPROVED** [review]

**Comment by daviroo** [comment]
Thanks for the contribution @eimolina

**Comment by eimolina** [comment]
I’m happy to help @daviroo. Thanks to you too, @dolcalmi; for advices.

**Pull request merged**